readme.md

```diff
@@ -0,0 +1,16 @@
+Welcome to the Documentation of our Technophobe project!
+
+Aim of the project: create a database of zoo animals, function, and api endpoint to manipulate this database.
```

Comment:
I would maybe be more specific about the functions, like "functions that create a database and import data on the animals"